common/main.conf

```diff
@@ -1,13 +1,18 @@
 # Log notice level and higher (e.g. state cache
 # emits useful log messages on notice level).
-error_log stderr notice;
+error_log stderr info;
 
 
 # Make env vars accessible from within Lua modules.
 env SECRET_KEY_FILE_PATH;
 env AUTH_ERROR_PAGE_DIR_PATH;
 env OAUTH_CLIENT_ID;
 env OAUTH_AUTH_REDIRECTOR;
+env DEFAULT_SCHEME;
+
+env FIRST_POLL_PERIOD_SECONDS;
+env POLL_PERIOD_SECONDS;
+env CACHE_EXPIRATION_SECONDS;
```

Comment:
+1 to CACHE_ - seems nice, but I am not sure if adding _SECONDS suffix is a good idea/it's rarely used in practice. It seems redundant - units should be documented in the code.

I have renamed it the way you suggested plus added a comment. I do not think it's worth having longer discussion about it :)